tests/internal/test_module.py

```diff
@@ -65,7 +80,10 @@ def test_import_origin_hook_for_module_not_yet_imported():
     path = os.getenv("MODULE_ORIGIN")
     hook = mock.Mock()
 
-    ModuleWatchdog.install()
+    try:
+        ModuleWatchdog.install()
+    except RuntimeError:
+        pass
```

Comment:
when/why could this happen? why do we not care?

Reply:
If `ModuleWatchdog` is already installed we get a `RuntimeError`. This makes the call to `install` idempotent

Reply:
🤔 since we install it by default should it always be available, do we need to change these tests to assume it is already available? or to use it's own instance of `ModuleWatchdog` instead of the globally installed one?

Reply:
> or to use it's own instance of ModuleWatchdog instead of the globally installed one?

The fixture should be removing the global one if it's installed and then reinstalling it at the end.

Reply:
ok, so if that is the case, then this shouldn't fail ever right?

Reply:
Ah, my bad! Here we are in a subprocess, where the fixture does not apply. That's why we have this pattern only in subprocess tests.

Reply:
oh, I missed that these were in subprocesses as well

so that should mean these are running in isolation then and can probably just assume that `ModuleWatchdog` is already installed